app/place/admin.py

```diff
@@ -1,6 +1,18 @@
-from django.contrib import admin
-
+from django.contrib import admin, messages
+from rollup.utils import load_rollup
+from django_object_actions import DjangoObjectActions
 from .models import Place
 
-# Register your models here.
-admin.site.register(Place)
+# Adds a button to the Person Admin page to allow users to manually update the
+# directory screens with the most up-to-date data
+class PlaceAdmin(DjangoObjectActions, admin.ModelAdmin):
```

Comment:
As this is essentially exactly the same as the class for the person admin, you could rename it to something generic in one of the two admin files, then import it into the other, just changing the `admin.site.register` method. Alternatively could be placed in a utils file. Seeing as it utilizes the `load_rollup` function, it might make sense for this to live in the `rollup.utils` folder, and be imported into both person and place admin files. 

What do you think @jonseitz ?

Reply:
I agree with @chrxr. Of the two approaches, I prefer the idea of having this "UpdateScreens" code live alongside the rollup and then importing/registering it with the two models separately.

Reply:
Thanks for the suggestions. I added `UpdateScreens` to the rollup.utils file in fa713dc.